GliaWidgets/Theme/Alert/AlertConfiguration.swift

```diff
@@ -13,6 +13,7 @@ public struct AlertConfiguration {
     public var operatorsUnavailable: MessageAlertConfiguration
 
     /// Configuration of the media upgrade confirmation alert.
+    @available(*, deprecated, message: "Not used")
```

Comment:
do we have another interface for setting up UI for mediaUpgrade alert?

Reply:
multiple media upgrade was never implemented, it's basically reduntant